.eslintrc.js

```diff
@@ -75,10 +75,7 @@ module.exports = {
         "key-spacing": "off",
         "keyword-spacing": "off",
         "line-comment-position": "off",
-        "linebreak-style": [
-            "error",
-            "unix"
-        ],
+        "linebreak-style": "off",
```

Comment:
Scary. What do we need this for?

Reply:
Because when git.autocrlf is (correctly) turned on for a Windows machine, it fails the linter.